src/routes/StackView/StackView.tsx

```diff
@@ -3,24 +3,45 @@ import {useDispatch} from "react-redux";
 import {useParams, useNavigate} from "react-router";
 import {useTranslation} from "react-i18next";
 import _ from "underscore";
+import {animated, Transition} from "react-spring";
 import {Color, getColorClassName} from "constants/colors";
 import {NoteDialogComponents} from "components/NoteDialogComponents";
 import {Portal} from "components/Portal";
 import {useAppSelector} from "store";
 import {Actions} from "store/action";
 import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
 import "./StackView.scss";
+import {StackNavigation} from "components/StackNavigation";
+import {useEffect, useRef, useState} from "react";
+import {Note} from "types/note";
+import {AvataaarProps} from "components/Avatar";
+
+type StackedNote = Note & {
+  authorName: string;
+  avatar?: AvataaarProps;
+};
+
+const getTransform = (state: "start" | "end", dir?: string) => {
+  if (dir === "left") {
+    return state === "start" ? "translateX(-100%)" : "translateX(100%)";
+  }
+  if (dir === "right") {
+    return state === "start" ? "translateX(100%)" : "translateX(-100%)";
+  }
+  return "translateX(0)";
```

Comment:
This could probably also be made more minimal (especially the left/right start/end part), but it will suffice.

Reply:
How tho? It's two variables with two possible values each, so four combinations

Reply:
My thought was to use either just two variables or map left/right and start/stop to 1/-1 and then just have a multiplication

Reply:
just pushed a version with mapping to 1/-1. Can you check if you find it more readable now?